include/staff/templates/status-options.tmpl.php

```diff
@@ -66,7 +66,7 @@ class="action-dropdown anchor-right">
                 ><i class="<?php
                         echo $actions[$status->getState()]['icon'] ?: 'icon-tag';
                     ?>"></i> <?php
-                        echo __($status->getName()); ?></a>
```

Comment:
The purpose of `__()` is to translate the given string. Please see:
- https://github.com/osTicket/osTicket/blob/develop/include/class.translation.php#L1053-L1058

Reply:
This maybe works with your default statuses which are in the transaltion files. But when you add own entries in the custom list *Ticket Statuses* this don't work because of the custom translation format:

https://github.com/osTicket/osTicket/blob/develop/include/class.list.php#L1271-L1273